### ISSUE TITLE
Scrolling on mobile

### ISSUE THREAD

**Issue by fezproof** [BeeeQueue/isthepatchout#301]
Hey great app! Love the stack, but have noticed that I cannot scroll on mobile to see the links at the bottom page.

Probably an easy css fix :)

**Comment by BeeeQueue** [comment]
It's a bug only in Safari, the worst web browser atm

**Comment by fezproof** [comment]
This happened on Chrome for iOS

**Comment by BeeeQueue** [comment]
Apple are garbage and don't allow any browser other than Safari on iOS. Any browser you _think_ you might have installed is actually just a fancy shell for Safari

**Comment by fezproof** [comment]
Yep, I’m aware... Just letting you know there is a bug with your css :)
